packages/node_modules/@webex/plugin-meetings/src/meeting-info/meeting-info-v2.js

```diff
@@ -77,6 +77,59 @@ export default class MeetingInfoV2 {
     });
   }
 
+  /**
+     * Fetches meeting info from the server
```

Comment:
The comments are not entirely correct. This function does more than just fetching meeting info, it creates an adhoc meeting

Reply:
done